inst/htmljs/animint.js

```diff
@@ -1626,11 +1627,22 @@ var animint = function (to_select, json_file) {
 	  return d["clickSelects.variable"] + " " + d["clickSelects.value"];
 	};
       }
-      // if elements have an existing title, remove it.
+      // Setting the data-tippy-content attribute on each element
+      elements.each(function (d) {
```

Comment:
please try elements.attr(text_fun) instead

(notice that .each is not used anywhere else in code)